extensions/wikia/RTE/RTE.class.php

```diff
@@ -689,7 +689,7 @@ static public function userGetOption($options, $name, &$value) {
 		$useEditor = $wgRequest->getVal('useeditor', false);
 
 		// do not continue if user didn't turn on wysiwyg in preferences
-		if ( ( $options['editor'] === '1' && empty( $useEditor ) )
+		if ( ( isset( $options['editor'] ) && $options['editor'] === '1' && empty( $useEditor ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)